src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -323,7 +323,26 @@
   <data name="ContainerIsRequiredForAnonymizedExport" xml:space="preserve">
     <value>Missing '_container' for anonymized export</value>
   </data>
+  <data name="ConvertDataParameterNotValid" xml:space="preserve">
+    <value>Data convert does not support the following parameter {0} for a POST operation.</value>
```

Comment:
>POST operation [](start = 75, length = 14)

nit: Does Converter has support anything other than POST, or plan to support in future? I find it a bit weird to specify here type of http method.

Reply:
Also curious, is there are much value in separating `ConvertDataParametersNotValid` and `ConvertDataParameterNotValid` ?

---
In reply to: [528900317](https://github.com/microsoft/fhir-server/pull/1434#discussion_r528900317) [](ancestors = 528900317)

Reply:
```ConvertDataParametersNotValid``` means the body is invalid (not a valid Parameters resource or added unsupported parameters)
While ```ConvertDataParameterNotValid``` means a specific Paramter entry is not valid. The difference is meaningful as we are using a [Parameters](http://hl7.org/fhir/parameters.html) resource?